### PR TITLE
Remember user who created a site.

### PR DESCRIPTION
**Pull request by diosmosis**

### CODE REVIEW

File: core/Db/Schema/Mysql.php [hunk 1]
```diff
@@ -75,6 +75,7 @@ public function getTablesCreateSql()
                             `group` VARCHAR(250) NOT NULL,
                             `type` VARCHAR(255) NOT NULL,
                             keep_url_fragment TINYINT NOT NULL DEFAULT 0,
+                            user_created VARCHAR(100) NULL,
```
Comment (tsteur) [inline core/Db/Schema/Mysql.php]: would it be `login_created` or `user_login_created` or `login_creator` or something? In the user table it is the `login` column that matches it. Might be more clear? not sure...

Comment (diosmosis) [reply core/Db/Schema/Mysql.php]: Can change it to `creator_login` as in "the creator's login"? Was trying to be consistent w/ the `ts_created` column w/ `user_created`.

Comment (tsteur) [reply core/Db/Schema/Mysql.php]: `creator_login ` 👍 I was actually just thinking before the same. otherwise sounds like a user was created

**Comment by diosmosis** [comment]
Updated.

File: core/Site.php [hunk 2]
```diff
@@ -636,4 +636,15 @@ public static function getExcludedQueryParametersFor($idsite)
...
         return self::getFor($idsite, 'excluded_parameters');
     }
+
+    /**
+     * Returns the user that created this site.
+     *
+     * @param int $idsite The site ID.
+     * @return string|null If null, the site was created before the creation user was tracked.
+     */
+    public static function getCreationUserFor($idsite)
```
Comment (tsteur) [inline core/Site.php]: Maybe `getCreatorLoginFor`? For consistency be also good to add a `public function getCreatorLogin`

**Comment by tsteur** [comment]
I presume tests need to be fixed but otherwise looks good to merge.

**Pull request merged**